portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,95 @@
-#content {
+.content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  margin-bottom: 20px;
+  width: 75%;
 }
 
-#greeting-container {
+.projpic {
+  width: 30%;
+  padding-left: 10px;
+  padding-right: 10px;
   margin-top: 20px;
+  margin-bottom: 20px;
+}
+.randomFact {
+  margin-top: 20px;
+  margin-bottom: 30px;
 }
 
-.profilePic {
+.profile{
+  margin-right: 20px;
+  float: left;
+  text-align: center;
+}
+
+#profilePic {
   display: block;
   margin-right: auto;
   margin-left: auto;
-  width: 40%;
+  width: 30%;
+  height: auto;
+}
+
+/* Add a black background color to the top navigation */
+.nav {
+  background-color: #333;
+  overflow: hidden;
 }
+
+/* Style the links inside the navigation bar */
+.nav a {
+  float: left;
+  display: block;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+/* Change the color of links on hover */
+.nav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+/* Add an active class to highlight the current page */
+.nav a.active {
+  background-color: #4CAF50;
+  color: white;
+}
+
+/* Hide the link that should open and close the nav on small screens */
+.nav .icon {
+  display: none;
+  font-size: 15px;
+}
+
+/* When the screen is less than 600 pixels wide, hide all links, 
+ * except for the first one ("Home"). 
+ * Show the link that contains should open and close the nav (.icon) */
+@media screen and (max-width: 600px) {
+  .nav a:not(:first-child) {display: none;}
+  .nav a.icon {
+    float: right;
+    display: block;
+  }
+}
+
+/* The "responsive" class is added to the nav with JavaScript 
+ * when the user clicks on the icon. This class makes 
+ * the nav look good on small screens (display the links vertically) */
```

Comment:
I encourage you to see if you can do this automatically via `@media` queries without using JavaScript. Conceivably in a separate CL.

Reply:
Sorry, what is CL?

Reply:
Additionally, how could I achieve the same vertical dropdown without using JavaScript? In my mind, I'm imagining two situations for the navigation bar on small screens:

1. The icon hides the link to other two pages when staying on the home page. In this case, only the "Home" and the icon should be visible on the screen, and other pages should be hidden. 
2. Navigation to other pages by first clicking the icon to open a vertical dropdown and then select the correct page to go to. In this case, the navigation to other pages should be visible, shown in block style. 

Because of this, I think I need a way to tell the site when to hide and show the links to other pages, and I thought JavaScript would be a good way to achieve this by making a change to the class name following the onclick event on the icon.

Reply:
I was mostly thinking that you'd have a "it looks good" layout on small screens enabled entirely by media query. I'm being previous though. I just have a preference for websites that can work without JavaScript. This is fine. We can nerd about script vs no-script later.